packages/babel-parser/test/fixtures/es2022/class-private-names-duplicated/static-block-duplicate-name/input.js

```diff
@@ -0,0 +1,6 @@
+class X {
```

Comment:
We can remove the other two tests in `es2015`, only the `es2022` one is sufficient.

Not related to this PR: I realize that the `static-block` test is still in the `experimental` folder, we should have moved it to the `es2022`.